### PR TITLE
Show detail

### PR DESCRIPTION
**Pull request by Falconiac**
Hi, please review our pr !!! Thank you !!!

### CODE REVIEW

File: src/components/EventCard/EventCard.jsx [hunk 1]
```diff
@@ -1,19 +1,38 @@
+import {useState} from 'react';
+
+import Detail from '../Detail/Detail';
+
 import EventCardContainer from './EventCard.styled';
 
 const EventCard = ({event}) => {
+	const [details, setDetails] = useState(false);
+	console.log(details);
```
Comment (F-Kirchhoff) [inline src/components/EventCard/EventCard.jsx:9]: Please remove all console.logs 🕵️

File: src/components/EventCard/EventCard.jsx [hunk 2]
```diff
@@ -1,19 +1,38 @@
...
-				<p className="description">{event.desc}</p>
-				<p className="date">{event.date}</p>
-			</article>
-		</EventCardContainer>
+		<>
+			<EventCardContainer
+				details={details}
+				onClick={() => {
+					setDetails(!details);
+				}}
```
Comment (F-Kirchhoff) [inline src/components/EventCard/EventCard.jsx:20]: You can use your function switcher here

Comment (Falconiac) [reply src/components/EventCard/EventCard.jsx]: Hey Felix, this is done...

File: src/components/EventCard/EventCard.jsx (hunk 1 above)
Comment (Falconiac) [reply src/components/EventCard/EventCard.jsx]: Hey Felix, this is done

File: src/components/Detail/Detail.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import DetailPOP from './Detail.styled';
+
+const Detail = ({event, back}) => {
+	return (
+		<DetailPOP className="background">
+			<article>
+				<button onClick={back}>X</button>
+				<h2>{event.eventName}</h2>
+				<p>{event.date}</p>
+				<p>{event.desc}</p>
```
[the remark this answers is not in the document]
Comment (Falconiac) [reply src/components/Detail/Detail.jsx]: Hey Felix, this is done

**Pull request merged**